utilities/validations.py

```diff
@@ -133,6 +133,14 @@ def validate_structure_id(**kwargs):
     table
     :param structure_id
     """
+    if(kwargs.get('check_unique_structure_id')):
+        for structure in kwargs['data']:
+            existing_structure = StructureModel.query.filter_by(
+                structure_id=structure['structure_id']).first()
+            if existing_structure:
```

Comment:
@jesseinit Have you considered validating for the scenario where a list of structures with the same structure_id is passed? We want to return a user-friendly message other than returning the `psycopg2.IntegrityError` from the DB.

Reply:
@ThaDeveloper Yeah that check is incorporated in the logic. If you try to run the mutation in the PR description twice you'd see that your concern has been addressed.

Reply:
I don't mean running the mutation twice, I mean adding two structures with similar Ids at the same mutation. I received an error as shown here. 
![image](https://user-images.githubusercontent.com/12385482/57769259-1c579b00-7716-11e9-9cde-fd4dfa65141f.png)

Reply:
@ThaDeveloper 
It has been addressed. Kindly update the branch and retry